server/series/models.py

```diff
@@ -62,3 +77,4 @@ class Status(models.TextChoices):
     team = models.ForeignKey(Team, on_delete=models.CASCADE)
     status = models.CharField(max_length=8, choices=Status.choices, default=Status.PENDING)
     expires_on = models.DateField(default=default_invitation_expiry_date)
+    rsvp_date = models.DateField(blank=True, null=True)
```

Comment:
can we add default as today?

Reply:
hmm yes, can do

Reply:
wait this is RSVP date, so we can't have a default right ? depends on when the player accepts no ?

Reply:
yeah lol i was confused when i first saw it